symfony/client/src/core/util/validation/rules.ts

```diff
@@ -422,7 +422,7 @@ export const shouldNotLessThanCharLength = function(charLength: number) {
  * @param {string | function} minValue
  * @param {string|undefined} message
  */
-export const minValueShouldBeLowerThanMaxValue = (
+export const maxValueShouldBeGreaterThanMinValue = (
```

Comment:
Chamara said to refactor this since the logic of the function is written for checking whether the max was greater than the min. I was previously using this rule for the max field on the page as well. So this name makes more sense.